CHANGELOG.md

```diff
@@ -219,14 +219,14 @@ introducing the preprocessor directive `NODE_API_SWALLOW_UNTHROWABLE_EXCEPTIONS`
 
 #### Documentation
 
-- Added docuemtnation for `Napi::Object::PropertyLValue`.
+- Added documentation for `Napi::Object::PropertyLValue`.
 - Changed all N-API references to Node-API.
 - Some minor corrections all over the documentation.
 
 #### TEST
 
-- Added tests relating to fetch property from Global Object.
-- Added addtiona tests for `Napi::Object`.
+- Added tests relating to fetching property from Global Object.
```

Comment:
I think this is about a `fetch` property and not about fetching properties, but I'm not sure. @nodejs/node-api 

```suggestion
- Added tests relating to `fetch` property from Global Object.
```